frontend/app/js/app.coffee

```diff
@@ -135,3 +138,50 @@ dismissTeamManager = (username, slug) ->
 ) window, document, "script", "//www.google-analytics.com/analytics.js", "ga"
 ga "create", "UA-47528450-1", "exercism.io"
 ga "send", "pageview"
+
+class SubmissionPrompt
+  constructor: (container) ->
+    @container = $(container)
+    @initializeUI()
+
+  initializeUI: ->
+    @container.find('.btn-answer-prompt').click ->
+      $('#submission_comment').focus()
+    @container.find('.btn-new-prompt').click => @showNewPrompt()
+    @showNewPrompt()
+    @container.removeClass('hidden') # Leave hidden when JS not enabled
+
+  showNewPrompt: ->
+    target = @container.find('.prompt-text')
+    fadeDuration = 50
+    target.fadeOut fadeDuration, =>
+      target.text @randomPrompt()
+      target.fadeIn fadeDuration
+
+  randomPrompt: ->
+    _.sample(@prompts)
```

Comment:
That's interesting since the library was already present. I wonder if it was previously used and the code was removed, or if it is a dependency of a dependency. Git blame is probably telling.